### DISCUSSION TITLE
Can’t connect to jellyfin server on other devices

### DISCUSSION THREAD

**Discussion by 7549brijesh** [jellyfin/jellyfin#7812] [Q&A] [answered]
Currently I am on endeavour os(desktop as a host machine) and trying to setup jellyfin 10.7 on it 192.168.XX.XX:8096 when tried to access in phone but unsuccessful(I wanted to access jellyfin in my local network) via wifi router

settings in jellyfin:

1.allow the remote access(done)

2.bind to local ip address to [0.0.0.0](done) but unsuccessful

3.tried port forwarding(but my mind said this is not needed if i am going to use in my local network)

setting that I did in endeavours os(linux):

1.allowed in/out bound(using gufw) in firewall to port 8096

Need a help??
still I can run it on host machine
![2022-05-29_09-30](https://user-images.githubusercontent.com/25589680/170857778-f82d33eb-5f72-46b6-9a5e-5d0f6c28b418.png)

**Comment by SR-Lut3t1um** [comment]
What do you mean by unsuccesfull? 
can u run this, to check what interface jellyfin is listening to?: 
`sudo ss -tulwn | grep LISTEN`

Also you said you allowed 8096 however in the image you've linked you're using 8196.

**Comment by Thomaz-bp** [comment, marked as answer]
**Fixed after 5 hours.** 
After a lot of digging finally solved it. Had some troubleshooting of my own. 
Here is what I found and my fast solution:
- I tried opening all ports and disabling firewall in ther server with no solution;
- I can connect in my own server through ip, but can't connect with another device in the same network. I first thought it was a problem with jellyfin, so I downloaded another server (xampp,nginx,wamp) and tried to connect with my local network, but had the same issue.
- My solution was to try another network, so I turned the "Hotspot" function and it created another virtual network, that way I fixed my problem and could connect with my phone.
- It was clear the problem was with my router, I had already done a lot but it was something I could netver imagine. I read this topic [https://community.tp-link.com/en/home/forum/topic/77845](url)
- The solution: change in router "Mode ( Wireless -> Wireless Settings ): **11bg mixed** (it was 11bgn mixed)"

Edit: some typos
